src/js/worker/core/player.js

```diff
@@ -727,15 +758,22 @@ function genRatings(
         season,
         skills: [],
     };
+	
+	if (predeterminedHeight !== undefined)
+		ratings.hgt = predeterminedHeight;
 
     // Ugly hack: Tall people can't dribble/pass very well
     if (ratings.hgt > 40) {
-        ratings.drb = limitRating(ratings.drb - (ratings.hgt - 50));
-        ratings.pss = limitRating(ratings.pss - (ratings.hgt - 50));
+        ratings.drb = limitRating(ratings.drb - (ratings.hgt - 40));
+        ratings.pss = limitRating(ratings.pss - (ratings.hgt - 40));
     } else {
         ratings.drb = limitRating(ratings.drb + 10);
         ratings.pss = limitRating(ratings.pss + 10);
     }
+	
+	// Most ultra-tall players have endurance issues
+	if (ratings.hgt > 90 && random.randInt(0,100) < 90)
+		ratings.endu = helpers.bound(ratings.endu, 0, 15);
```

Comment:
True.  How would you do it?  Injuries are the bigger issue, actually.  I haven't looked at the injury subprograms yet, but doubling or tripling the injury rate for hgt 90+ may be a better approach and just drop this.